psydac/api/grid.py

```diff
@@ -230,11 +230,12 @@ def space(self):
 # TODO have a parallel version of this function, as done for fem
 def create_collocation_basis( glob_points, space, nderiv=1 ):
 
-    T    = space.knots      # knots sequence
-    p    = space.degree     # spline degree
-    n    = space.nbasis     # total number of control points
-    grid = space.breaks     # breakpoints
-    nc   = space.ncells     # number of cells in domain (nc=len(grid)-1)
+    T     = space.knots      # knots sequence
+    p     = space.degree     # spline degree
+    n     = space.nbasis     # total number of control points
+    grid  = space.breaks     # breakpoints
+    nc    = space.ncells     # number of cells in domain (nc=len(grid)-1)
+    perio = space.perio
```

Comment:
`perio` ? Several FemSpace subclasses have a `periodic` attribute, but I could not find any `perio`.

Also, a docstring would be nice :)

Reply:
You're right but it looks like this function is actually never called and I don't know what it's suppose to do.

Reply:
that's already something to write in the docstring 😅

Reply:
that's already something to write in the docstring 😅